src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1142,19 +1082,10 @@ def load_pem_public_key(self, data):
     def load_der_private_key(self, data, password):
         # OpenSSL has a function called d2i_AutoPrivateKey that can simplify
         # this. Unfortunately it doesn't properly support PKCS8 on OpenSSL
-        # 0.9.8 so we can't use it. Instead we sequentially try to load it 3
+        # 0.9.8 so we can't use it. Instead we sequentially try to load it 2
```

Comment:
We might be able to switch to d2i_AutoPrivateKey here now, but that should be a separate PR.